satellite/api.go

```diff
@@ -42,32 +41,9 @@ import (
 	"storj.io/storj/satellite/orders"
 	"storj.io/storj/satellite/overlay"
 	"storj.io/storj/satellite/repair/irreparable"
-	"storj.io/storj/satellite/repair/repairer"
 	"storj.io/storj/satellite/vouchers"
 )
 
-// APIConfig is the global config for the satellite API process
```

Comment:
why did we decide to remove this separate api config?

Reply:
for simplicity, i chose to use the same Peer config, however at some point in the future, we may want separate configs.